src/main/java/edu/kis/vh/nursery/ListIntegers/IntLinkedList.java

```diff
@@ -1,10 +1,27 @@
-package edu.kis.vh.nursery.list;
+package edu.kis.vh.nursery.ListIntegers;
+
+import edu.kis.vh.nursery.ListInterface;
+
+public class IntLinkedList implements ListInterface {
+
+	private class Node {
+
+		final private int value;
+		private Node prev;
```

Comment:
You should perform encapsulation here (generate getters and setters). See "accessory i hermetyzacja"